applications/hotornot/src/hon_rater.erl

```diff
@@ -95,6 +99,17 @@ get_rate_data(JObj, ToDID, FromDID, Rates) ->
             {'ok', rate_resp(Rate, JObj)}
     end.
 
+-spec maybe_add_resource_flag(kz_json:object(), ne_binary()) -> kz_proplist().
+maybe_add_resource_flag(JObj, AccountId) ->
+    case kapps_account_config:get_from_reseller(AccountId, ?APP_NAME, <<"filter_by_resource_id">>, 'false') of
+        'true' ->
+            case kz_json:get_value(<<"Resource-ID">>, JObj) of
+                'undefined' -> [];
+                ResourceId -> [ResourceId]
```

Comment:
@onnet i missed the hon_util part that does the filtering. previous was more readable but not being a permanent fix, changing the signature of the funs doesn't seem appropriate.

the previous comment is still valid if you handle the tuple in `hon_util:matching_options` i think